components/crud-web-apps/jupyter/frontend/src/app/pages/form/form-default/form-image/form-image.component.html

```diff
@@ -4,8 +4,23 @@
   ML packages"
   icon="fa:fab:docker"
 >
+  <section>
+    <div class="server-type">Choose server type:</div>
+    <mat-button-toggle-group [formControl]="parentForm.get('serverType')" aria-label="Server Type">
+      <mat-button-toggle value="jupyter" aria-label="Use JupyterLab based server">
+        <mat-icon style="height:72px !important; width:200px !important;" svgIcon="jupyterlab"></mat-icon>
```

Comment:
Don't modify html elements directly with styles, always use a class instead that wraps what the style changes are. Also try to avoid as much as possible `!important` in CSS. Instead try to use more specific css selectors.

Here you could introduce and use the following CSS class for each `<mav-icon>`
```css
.server-type {
  height: 32px;
  width: 150px;
}
```